api/src/resources/dataset.rs

```diff
@@ -185,3 +227,54 @@ pub(crate) struct UpdateRequest<'request> {
 pub(crate) struct UpdateResponse {
     pub dataset: Dataset,
 }
+
+#[cfg(test)]
+mod tests {
+    use crate::resources::comment::CommentTimestampFilter;
+
+    use super::*;
+    use chrono::TimeZone;
+    use serde_json::json;
+
+    #[test]
+    pub fn test_serialize_statistics_request_params_default() {
+        let params = StatisticsRequestParams {
+            ..Default::default()
+        };
+
+        assert_eq!(
+            json!(params).to_string(),
```

Comment:
You're doing an extra step here of `struct` -> `serde_json::Value` -> `String`. You can use `serde_json::to_string` to convert straight to a `String`.

Note this has the slightly different behaviour that the fields are serialized in the order they are defined in the `struct` - so I've sorted the struct fields for consistent behaviour.